nepalingo-web/src/components/userAuth/UserAuth.tsx

```diff
@@ -130,7 +130,7 @@ const User_auth: React.FC = () => {
           </div>
         </div>
 
-        {action === "Log In" ? null : (
+        {action === "Sign Up" ? null : (
           <div className="text-right text-sm text-gray-500 mb-7">
```

Comment:
THis is good for now, I just wanted to give you a tip for the future - you can use conditionals in JSX. See here: https://react.dev/learn/conditional-rendering